src/modules/alsa.cpp

```diff
@@ -261,6 +262,7 @@ namespace modules {
     }
     const auto& mixers = get_mixers();
     for (auto&& mixer : mixers) {
+      mixer->set_mute(m_unmuteOnScroll || !mixer->is_muted());
```

Comment:
Wouldn't this mute instead of unmute?

In any case, we should only call the `set_mute` method if `unmute-on-scroll` is enabled, to avoid the overhead of calling into the alsa library

Reply:
I did check this before because it confused me as well, but no. Alsa's set_mute is inverted, meaning passing false makes it muted.
I verified (Before and after the incoming fixes) both modules on my computer, and muting and unmuting work correctly.

Reply:
Well that's incredibly bad design on our part, we will have to fix that at some point